Ruby/Ruby.sublime-syntax

```diff
@@ -157,8 +157,8 @@ contexts:
       pop: true
 
   module:
-    - match: \b(module)\b
-      scope: meta.module.ruby keyword.control.module.ruby
+    - match: \bmodule\b
+      scope: meta.module.ruby storage.type.module.ruby
```

Comment:
`keyword.declaration.module` imo.

Reply:
I was thinking about `storage.type.namespace` as a module can be compared to a namespace, I think, but in general most syntaxes use `storage.type.namespace` for this kind of syntax, and so I perefer here.

Reply:
Same argument as for classes.